artifactory/commands/generic/upload.go

```diff
@@ -56,6 +78,16 @@ func (uc *UploadCommand) Run() error {
 // Uploads the artifacts in the specified local path pattern to the specified target path.
 // Returns the total number of artifacts successfully uploaded.
 func (uc *UploadCommand) upload() error {
+	//in case of sync-delete ger user confirm first and save the operation timestamp.
+	syncDeleteProp := ""
+	if !uc.DryRun() && uc.SyncDeletePath() != "" {
+		if uc.quiet || cliutils.InteractiveConfirm("Sync-delete may delete some artifacts. Are you sure you want to continue?") {
+			timestamp := strconv.FormatInt(time.Now().UnixNano()/int64(time.Millisecond), 10)
```

Comment:
Yes. it's the same.
**java's System.currentTimeMillis():**
This method returns the difference, measured in milliseconds, between the current time and midnight, January 1, 1970 UTC(coordinated universal time).

**go's time.Now().UnixNano():**
UnixNano returns t as a Unix time, the number of nanoseconds elapsed since January 1, 1970 UTC.
Then we convert nano to mili by dividing in time.Millisecond